README.md

```diff
@@ -52,18 +52,36 @@
 
 ## Contributing
 
-## License
+Arrow-meta project is split into 4 packages:
+- compiler-plugin: Kotlin compiler plugin
+- gradle-plugin: Plugin for Gradle for activating Kotlin compiler plugin
+- idea-plugin: Plugin for Intellij IDEA for IDE integration (icons, autocomplete etc.)
+- testing-plugin: Tools for testing compiler and idea plugin
```

Comment:
Currently, the dsl to create idea plugin tests is in `idea-plugin`.
But is also not merged into master, yet 😄. 
We heavily rely dependencies which we already get from the existing set up in the `idea-plugin`.